wiki/Welcome/hu.md

```diff
@@ -1,130 +1,152 @@
 
-3. lépés - Töltsd be az osu!-ba!
---------------------------------
+![][welcome02 image]
 
-![".osz" fájl, tiszta beatmap fájl](Osz icon.png "fig:".osz" fájl, tiszta beatmap fájl") Amint a letöltésed befejeződött, megnyithatod a **.osz** kiterjesztésű fájlt. Ha ez valamiért nem tetszik, akkor be is húzhatod az osu! ablakába.
+**Megjegyzés:** Nem számít hová mented a fájlt amíg megtalálod miután befejeződött a letöltés. Egyébként a megnyithatod egyből, de ez el fogja indítani az osu!-t, ha az még nem fut.
 
-<img src="welcome03.png" title="welcome03.png" alt="welcome03.png" width="580" />
+### Harmadik lépés - Töltsd be az osu!-ba!
+![".osz", tiszta beatmap fájl][.osz icon link]
 
-4. lépés - Játssz szórakozásból vagy versengj másokkal!
--------------------------------------------------------
+Amint a letöltésed befejeződött, egyszerűen csak nyisd meg a **.osz** kiterjesztésű fájlt. Ha ez valamiért nem tetszik, akkor be is húzhatod az osu! ablakába.
 
-Amint megnyitod a fájlt, elindul az osu! (esetleg frissít is). Az elérhető beatmapjeid közötti válogatáshoz kattints a nagy osu! körre középen, majd válaszd a "Play" illetve a "Solo" lehetőséget. Láthatod is, hogy az új beatmap (a tutorial mellett) megjelent. Válassz nehézségi szintet és már játszhatsz is! A menüben "Solo" helyett a "Multi" lehetőséget is választhatod, ez a többjátékos mód, ahol játékosok egymás ellen versenyeznek élőben. Bármelyik nyitott játékhoz csatlakozhatsz (persze zárthoz is, ha tudod a jelszót).
+![][welcome03 image]
 
-<img src="welcome04.png" title="welcome04.png" alt="welcome04.png" width="580" />
+### Negyedik lépés - Játssz szórakozásból vagy versengj másokkal!
+Amint megnyitod a fájlt, elindul az osu! (esetleg frissít is). Az elérhető beatmap-jeid közötti válogatáshoz kattints a nagy osu! körre középen, majd válaszd a "Play" illetve a "Solo" lehetőséget. Az új beatmap (a tutorial mellett) kellett, hogy megjelenjen. Válassz nehézségi szintet, és már úton is vagy, hogy osu! játékossá válj! A menüben "Solo" helyett a "Multi" lehetőséget is választhatod, ez a többjátékos mód, ahol játékosok egymás ellen versenyeznek élőben. Vegyél részt te is a szórakozásban!
 
-Meg is vagyunk! Most már készen állsz az osu!-val való játékra és a kedvenc zenéid megkeresésére. Ha problémába ütközöl, nyugodtan kérj segítséget a fórumon, vagy a játékon belül a chatben az F8 gomb lenyomásával (a magyar közösség csatornája a \#Hungarian).
+![][welcome04 image]
 
-**SOK SZERENCSÉT ÉS JÓ SZÓRAKOZÁST! :3**
+Meg is vagyunk! Most már készen állsz, hogy te legyél a következő osu! bajnok! Ha bármilyen problémába ütközöl, nyugodtan kérj segítséget a fórumon, vagy a játékon belül (lenyomva az F8 gombot a chat panelhez és a \#help-et választva (a magyar közösség csatornája a \#hungarian).
 
-Játékmódok
-==========
+**SOK SZERENCSÉT ÉS JÓ SZÓRAKOZÁST!**
```

Comment:
🤔 ... I'll leave it up to you if you want to keep it. But if you do keep it, please use sentence casing:
```
**Sok szerencsét és jó szórakozást!**
```